index.html

```diff
@@ -437,6 +437,9 @@ <h3>Email</h3>
           <li>
             Design: <a href="http://html5up.net" target="_blank">HTML5 UP</a>
           </li>
+          <li>

```

Comment:
Can you please change the text from "Source at GitHub" to "Source: GitHub". I realized it's better to make things consistent.